gfx/map/map_object_data/generated/steppe_bush_01_generator.txt

```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
 	layer="grass_layer"
```

Comment:
> Provincial lines are incorrect for Darkshore.

And for Azshara.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132693780-f859ae69-3402-4d1a-a02e-b4b611fe9b85.png)
![image](https://user-images.githubusercontent.com/46384992/132693797-fcfdcd7b-9354-4cea-8cdc-e6e9c5dfa68f.png)

</details>

What do you want to change to fix it? Height and texture map or province map?

Reply:
_Provincial map. Azshara I am planning to upgrade in a separate issue but I will note that in this PR under Known Issues._

Reply:
> _Provincial map. Azshara I am planning to upgrade in a separate issue but I will note that in this PR under Known Issues._

On what you based height map?

Reply:
So I would move the Darkshore coastline back, changing shape if you want.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132877806-f4c814d2-c227-4bb5-8647-1697d1011b58.png)

</details>

Our Kalimdor is wider and it breaks proportions.